Sources/GRPC/HTTP1ToRawGRPCServerCodec.swift

```diff
@@ -277,8 +326,19 @@ extension HTTP1ToRawGRPCServerCodec: ChannelOutboundHandler {
         context.write(self.wrapOutboundOut(.end(trailers)), promise: promise)
       }
 
-      outboundState = .ignore
-      inboundState = .ignore
+      // Log the call duration and status
+      if let stopwatch = self.stopwatch {
+        self.stopwatch = nil
+        let millis = stopwatch.elapsedMillis()
+
+        self.accessLog.info("rpc call finished", metadata: [
+          "duration_ms": "\(millis)",
```

Comment:
I personally prefer `String(describing: )` in these cases over string interpolation, but feel free to keep this; whatever you prefer.

Reply:
We'd have to wrap it in `.string()` since it's logging metadata (i.e. the dictionary value type isn't `String`). String interpolation is the preferred method way here, see: https://github.com/apple/swift-log/blob/e8aabbe95db22e064ad42f1a4a9f8982664c70ed/Sources/Logging/Logging.swift#L303

Reply:
Oops, sorry! Thanks for pointing this out!